include/seqan3/range/views/minimiser.hpp

```diff
@@ -38,20 +38,9 @@ namespace seqan3::detail
  *
  * \details
  *
- * Note that most members of this class are generated by ranges::view_interface which is not yet documented here.
+ * See seqan3::views::minimiser for a detailed explanation on minimizers.
```

Comment:
You can, additionally add a `\sa seqan3::views::minimiser` if you deem it worth it.

Reply:
I added it